src/gui/optionsdlg.cpp

```diff
@@ -344,9 +344,20 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     connect(m_ui->spinMaxActiveDownloads, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
     connect(m_ui->spinMaxActiveUploads, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
     connect(m_ui->spinMaxActiveTorrents, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
-    connect(m_ui->checkIgnoreSlowTorrentsForQueueing, &QAbstractButton::toggled, this, &ThisType::enableApplyButton);
+    connect(m_ui->checkIgnoreSlowTorrentsForQueueing, &QGroupBox::toggled, this, &ThisType::enableApplyButton);
+    connect(m_ui->spinDownloadRateForSlowTorrents, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
+    connect(m_ui->spinUploadRateForSlowTorrents, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
+    connect(m_ui->spinSlowTorrentInactivityTimer, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
     connect(m_ui->checkEnableAddTrackers, &QGroupBox::toggled, this, &ThisType::enableApplyButton);
     connect(m_ui->textTrackers, &QPlainTextEdit::textChanged, this, &ThisType::enableApplyButton);
+
+    const QString slowTorrentsExplanation = QLatin1String("<html><body><p>")
+            + tr("A torrent will be considered slow if its download and upload rates stay below these values for &rdquo;Torrent inactivity timer&rdquo; seconds")
```

Comment:
html isn't required here

Reply:
I think, @Chocobo1 meant you don't need to enclose this string with html tags (html, body, p).

Reply:
But then it would look like this?

![capture](https://user-images.githubusercontent.com/6451685/35969745-884fb628-0cd0-11e8-9f2b-36ea6d1defdf.JPG)

Reply:
Oops... then leave it as is.

Reply:
>Oops... then leave it as is.

Okay.